src/main/java/core/basesyntax/service/CarServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.CarDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.lib.Service;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Driver;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).get();
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        carDao.update(car).getDrivers().add(driver);
```

Comment:
we should first modify the car and then call method update, not vice versa

Reply:
done